spec/unit/models/project_spec.rb

```diff
@@ -40,4 +40,51 @@
       end
     end
   end
+
+  describe '#blueprint' do
+    let(:project_data) do
+      { 'project' => {
+          'links' => { 'self' => '/gdc/foo' },
+          'meta' => { 'title' => 'My Project' } } }
```

Comment:
Style/MultilineHashBraceLayout: Closing hash brace must be on the line after the last hash element when opening brace is on a separate line from the first hash element.